*** dstat.orig	2021-09-14 20:25:22.226019108 +0900
--- dstat	2021-09-14 20:25:33.634150635 +0900
***************
*** 1,4 ****
! #!/usr/bin/env python
  
  ### This program is free software; you can redistribute it and/or
  ### modify it under the terms of the GNU General Public License
--- 1,4 ----
! #!/usr/bin/python3
  
  ### This program is free software; you can redistribute it and/or
  ### modify it under the terms of the GNU General Public License
